mmaction/models/backbones/x3d.py

```diff
@@ -0,0 +1,521 @@
+# pylint: disable=no-name-in-module
+# isort:maintain_block
+import math
+
+import torch.nn as nn
+import torch.utils.checkpoint as cp
+from mmcv.cnn import (ConvModule, Swish, build_activation_layer, constant_init,
+                      kaiming_init)
+from mmcv.runner import load_checkpoint
+from mmcv.utils import _BatchNorm
+
+from ...utils import get_root_logger
+from ..registry import BACKBONES
+
+# pylint: enable=no-name-in-module
+# isort:end_maintain_block
+
+
+class SEModule(nn.Module):
+
+    def __init__(self, channels, reduction):
+        super(SEModule, self).__init__()
+        self.avg_pool = nn.AdaptiveAvgPool3d(1)
+        self.bottleneck = self._round_width(channels, reduction)
+        self.fc1 = nn.Conv3d(
```

Comment:
I think calling it `fc` is better: 1. The layer is actually an `fc` layer, the feature map it operates on is of the size 1x1x1. Using `conv3d` to implement it can save 2 additional reshape. 2. In both SE-ResNet and the SlowFast repo, people call it `fc`.

Reply:
Argee